packages/components/src/styles/components/form/layout.scss

```
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM LAYOUT COMPONENTS
+//
+
+// FORM
+
+// Default max-width for form sections & similar components
+$hds-form-section-max-width: 768px;
+
+.hds-form {
+  --hds-form-section-max-width: #{$hds-form-section-max-width};
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+
+  // Set a max-width for the ButtonSet when it's within an Hds::Form
+  // Q: Should this be done (max-width on ButtonSet) and if so should it be done here or in the ButtonSet component?
+  .hds-button-set {
+    max-width: var(--hds-form-section-max-width, #{$hds-form-section-max-width});
+  }
+}
+
+// FORM HEADER & SECTION HEADER
+
+.hds-form__header,
+.hds-form__section-header {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+// Note: Only the Form Header needs a max-width as the Section Header is used within a Section
+.hds-form__header {
+  max-width: var(--hds-form-section-max-width, #{$hds-form-section-max-width});
+}
+
+// hasMaxWidth = false
+.hds-form__header--has-max-width-false {
+  --hds-form-section-max-width: none;
+}
+
+// FORM SECTION
+
+.hds-form__section {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  max-width: var(--hds-form-section-max-width, #{$hds-form-section-max-width});
+}
+
+// hasMaxWidth = false
+.hds-form__section--has-max-width-false {
+  --hds-form-section-max-width: none;
+}
```

[thought] what if instead we do this? it seems more semantically correct, no?

```suggestion
.hds-form__section {
  display: flex;
  flex-direction: column;
  gap: 24px;
}

.hds-form__section--has-max-width {
    max-width: var(--hds-form-section-max-width, #{$hds-form-section-max-width});
}
```

---

I already updated everything to work like this based on your earlier feedback regarding the class name.